docs/resources/rds_instance.md

```diff
@@ -163,6 +163,9 @@ The following arguments are supported:
 * `param_group_id` - (Optional, String, ForceNew) Specifies the parameter group ID. Changing this parameter will create
   a new resource.
 
+* `collation` - (Optional, String, ForceNew) Specifies the Character Set, only available to Microsoft SQL Server DB instances.
```

Comment:
as we does not set it, should we add this parameter into ignore  changes when importing?